validator/src/lib.rs

```diff
@@ -19,6 +19,7 @@ use {
 pub mod admin_rpc_service;
 pub mod bootstrap;
 pub mod cli;
+pub mod commands;
```

Comment:
I realize all of the other modules are `pub` here, but I think they could/should probably be `pub(crate)`. So, let's make the `commands` module `pub(crate)` and then we can reduce visibility on the other ones in the future